java/vector/src/main/java/org/apache/arrow/util/AutoCloseables.java

```diff
@@ -0,0 +1,204 @@
+/*
+ * Portions Copyright (C) 2017-2018 Dremio Corporation
```

Comment:
Is this intentional to have in the license?

Reply:
We generally preserve copyrights in straight-up copied files, it should be mentioned in LICENSE.txt though

Reply:
ok thanks, I wasn't sure what the case was here, so just thought I'd ask